docs/adnetwork.json

```diff
@@ -12,7 +12,9 @@
           "Rewarded",
           "Offerwall"
         ],
-        "infoPlistKey": ""
+        "infoPlistKey": "",
+        "unityAdapterName": "AMRAdapterAdcolony",
```

Comment:
adapter isimleri ios ve unity için aynı, o yüzden "unityAdapterName" yerine "adapterName" yapalım @Tahatosun

Reply:
Değiştirdim. @mhmtkrgz